src/core_ocean/mode_forward/mpas_ocn_time_integration_rk4.F

```diff
@@ -726,12 +727,13 @@ subroutine ocn_time_integrator_rk4(domain, dt)!{{{
          !$omp end do
 
          ! Compute normalGMBolusVelocity and the tracer transport velocity
-         if (config_use_standardGM) then
-             call ocn_gm_compute_Bolus_velocity(diagnosticsPool, meshPool, scratchPool)
+         if (config_use_GM) then
+             call ocn_gm_compute_Bolus_velocity(statePool, diagnosticsPool, &
```

Comment:
In rk4, the corresponding lines here should also be removed, since it is called from the driver.